test/hotspot/gtest/runtime/test_os.cpp

```diff
@@ -360,6 +360,19 @@ TEST_VM(os, jio_snprintf) {
 //  (from multiple calls to os::reserve_memory)
 static address reserve_multiple(int num_stripes, size_t stripe_len) {
   assert(is_aligned(stripe_len, os::vm_allocation_granularity()), "Sanity");
+
+#ifdef __APPLE__
+  // Workaround: try reserving memory with executable=true
+  // to figure out if that's supported on this macOS version
+  bool executable = true;
+  size_t test_len = 128;
+  address p_test = (address)os::reserve_memory(test_len, executable);
```

Comment:
Nit: if you just declare `p_test` as a `char*` then no casts needed here or on line 372.

Reply:
I like that, done.